app/controllers/admin/userManagement.py

```diff
@@ -100,20 +100,34 @@ def getProgramInfo(programID):
 @admin_bp.route('/admin', methods = ['GET'])
 def userManagement():
     terms = selectSurroundingTerms(g.current_term)
-    currentPrograms = (
+
+    if g.current_user.isCeltsAdmin:
+        currentPrograms = (
+                Program
+                .select(
+                    Program,
+                    fn.GROUP_CONCAT(fn.COALESCE(fn.CONCAT(User.firstName, ' ', User.lastName, '#', User.username), '')).alias('managers')
+                )
+                .join(ProgramManager, JOIN.LEFT_OUTER, on=(Program.id == ProgramManager.program))
+                .join(User, JOIN.LEFT_OUTER, on=(ProgramManager.user == User.username))
+                .group_by(Program.id)
+        )
+    
+    elif g.current_user.isProgramManager():
+        currentPrograms = (
             Program
             .select(
                 Program,
                 fn.GROUP_CONCAT(fn.COALESCE(fn.CONCAT(User.firstName, ' ', User.lastName, '#', User.username), '')).alias('managers')
             )
-            .join(ProgramManager, JOIN.LEFT_OUTER, on=(Program.id == ProgramManager.program))
+            .join(ProgramManager, JOIN.INNER, on=(Program.id == ProgramManager.program))
             .join(User, JOIN.LEFT_OUTER, on=(ProgramManager.user == User.username))
+            .where(ProgramManager.user == g.current_user.username)
             .group_by(Program.id)
-    )
```

Comment:
You can re-use this query instead of duplicating it in the different cases. The only real difference is the WHERE clause added to the second case (since the WHERE clause effectively turns the left join into an inner join automatically). So you can create the query, and if they are a program manager add the where clause to it.